pages/pipelines/build_timeouts.md

```diff
@@ -0,0 +1,38 @@
+# Build timeouts
+
+Build timeouts are limits on the maximum time a job can wait before being picked up by an agent. If a job exceeds the time limit, the job is automatically canceled and the build fails.
```

Comment:
Please check this. I think the page should start by saying what the thing is, so I added this part.